src/mon/MonCommands.h

```diff
@@ -704,6 +704,7 @@ COMMAND("osd set-nearfull-ratio " \
 	"name=ratio,type=CephFloat,range=0.0|1.0", \
 	"set usage ratio at which OSDs are marked near-full",
 	"osd", "rw", "cli,rest")
+COMMAND("osd get-require-min-compat-client", "get the minimum client version we will maintain compatibility with", "osd", "r", "cli,rest")
```

Comment:
I would appreciate if you could break this line to conform with the rest of the file, as well as the coding style guidelines.

Reply:
What's the idea behind the continuation slashes? I can't seem to find consistency when I look at other COMMANDs.